integration/travis/run_integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ev
+
+export PROJECT_DIR=`pwd`
+$PROJECT_DIR/../travis/start_scheduler.sh
+sleep 60
```

Comment:
FYI @dposada  - I believe that the failure you see there was a temporarily inability to download the public python 3 docker image.  Probably temporary network/bandwidth issues, I see that happen on Travis from time to time when downloading dependencies.  It's kind of funny because in this particular run, the process was able to download the same docker image later on in the test run (it's just the attempt to pre-load it that failed), so the actual simulation ran to completion normally...